contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -463,7 +467,7 @@ zfs_key_config_load(pam_handle_t *pamh, zfs_key_config_t *config,
 	if (!entry) {
 		free(config->runstatedir);
 		free(config->homes_prefix);
-		return (-1);
+		return (-2);
```

Comment:
bikeshedding: probably a good idea to make return codes here nicer, but how exactly? enum or defines? at the top of the file?